docs/data/material/getting-started/overview/overview.md

```diff
@@ -0,0 +1,26 @@
+# Material UI - Overview
+
+<p class="description">Material UI is library of React UI components that implements Google's Material Design.</p>
+
+## Introduction
+
+Material UI is an open-source React component library that implements Google's [Material Design](https://material.io/).
+
+It includes a comprehensive collection of prebuilt components that are ready for use in production right out of the box.
+
+Material UI is beautiful by design, and features a suite of customization options that make it easy to implement your own custom design system on top of our components.
+
+## Advantages of Material UI
+
+- **Ship faster:** thousands of open-source contributors have poured countless hours into these components. Focus on your core business logic—we've got your UI covered.
+- **Beautiful by default:** we're meticulous about our implementation of Material Design, ensuring that every Material UI component meets the highest standards of form and function.
```

Comment:
```suggestion
- **Beautiful by design:** we're meticulous about our implementation of Material Design, ensuring that every Material UI component meets the highest standards of form and function.
```
? (Although I guess "default" hints to the possibility of customising them...)

Reply:
I like it both ways. I guess I landed on "default" because it suggests that your app will look great even if you don't do any customizing.